R/stubs.R

```diff
@@ -24,5 +24,73 @@ sourceMarkers <- function(name, markers, basePath = NULL,
 
 #' @export
 navigateToFile <- function(file, line = 1L, column = 1L) {
-  callFun("navigateToFile", file, as.integer(line), as.integer(column))  
+  callFun("navigateToFile", file, as.integer(line), as.integer(column))
+}
+
+#' Replace Text within a Document
+#'
+#' Use this to change the contents of an open document.
+#'
+#' @param ranges A list of ranges. Each range is a four-element
+#'   integer vector, defining a document range with coordinates:
+#'
+#'   \code{startRow, startColumn, endRow, endColumn}
```

Comment:
I think this would be more user friendly with as an S3 object with simple constructor and print method. Would be nice to be able to use sentinel values (Inf?) to refer to last row/col